hack/validate-prow-job-semantics.py

```diff
@@ -132,6 +132,10 @@ def validate_names(path, data):
                     print("[INFO] {}: ci-operator job {} is ignored because of a label says so".format(path, job["name"]))
                     continue
 
+                if ("labels" in job) and ("ci-operator.openshift.io/prowgen-controlled" in job["labels"]) and job["labels"]["ci-operator.openshift.io/prowgen-controlled"] == "true":
+                    print("[INFO] {}: ci-operator job {} is ignored because it's generated and assumed to be right".format(path, job["name"]))
```

Comment:
Hmm, I'd be more comfortable if we had a test in `ci-tools` that guaranteed this.